dashboard/app/models/concerns/curriculum/course_types.rb

```diff
@@ -40,4 +46,16 @@ def can_be_participant?(user)
 
     false
   end
+
+  # A course is a professional learning if the participant audience is something
+  # other that students and therefore teaches adults
+  #
+  # This is different than courses that use the professional learning course models
+  # those can be checked for using old_professional_learning_course?
```

Comment:
Mostly curious, are the old professional learning course models now deprecated?  Do we expect to eventually get rid of the old professional learning course model entirely?

Reply:
The old professional learning course models are not completely deprecated because we have not built the peer review features they have into the normal course model. Deeper learning courses for facilitators need those features so for now deeper learning will continue to use those models as well as old courses however any new professional learning course that does not use peer review should be built in the normal course models.